ui/tests/integration/components/page/error-test.js

```diff
@@ -55,4 +56,29 @@ module('Integration | Component | page/error', function (hooks) {
         .hasText(this.error.errors[index], 'Error detail renders');
     });
   });
+
+  test('it should handle api client errors', async function (assert) {
+    this.error = getErrorResponse();
+
+    await render(hbs`<Page::Error @error={{this.error}} />`);
+
+    assert
+      .dom('[data-test-page-error-title]')
+      .hasText('404 Not Found', 'Error title renders based on status');
+    assert
+      .dom('[data-test-page-error-subtitle]')
```

Comment:
I think you added this one in the component, can you also add it to the GENERAL selector title [here](data-test-page-error-subtitle)?

Reply:
That link 404's but I added it to `tests/helpers/general-selectors.ts` file under the `pageError` object.